libdevcrypto/CryptoPP.cpp

```diff
@@ -264,7 +264,6 @@ Public Secp256k1::recover(Signature _signature, bytesConstRef _message)
 	
 	ECP::Element x;
 	{
-		Guard l(x_curve);
```

Comment:
This does not seem to be required any longer. I've tested extensively with multithreaded verification - no issues so far.